### ISSUE TITLE
Adding rules dynamically doesn't work when you reattach the sheet

### ISSUE THREAD

**Issue by AlastairTaft** [cssinjs/jss#438]
For example the following code works when I add rules to an already attached sheet.

```js
sheet.addRules({
	quoteNavigation: {
		backgroundColor: 'purple',
	},
	quoteNavigationMobile: {
		backgroundColor: 'red',
	},
})
```

But if I then run this line the css rules aren't present.

```js
sheet.detach().attach()
```

**Comment by kof** [comment]
This should have worked, can you submit a failing test to tests/functional/rules? do you mean a specific browser?

**Comment by kof** [comment]
ping @AlastairTaft

**Comment by AlastairTaft** [comment]
Sorry haven't had time to submit a test for this. For what it's worth, I found the issue while developing on chrome.

**Comment by kof** [comment]
Ok I got a failing test

**Comment by kof** [comment]
fixed in 7.0.3
